api/src/main/java/com/microsoft/gctoolkit/jvm/JavaVirtualMachine.java

```diff
@@ -69,15 +70,21 @@
 
     /**
      * Return the time of the first event in the GC log file.
-     * @return The time of the first event.
+     * @return The time of the last event.
      */
     DateTimeStamp getTimeOfFirstEvent();
 
+    /**
+     * Return the time of the first event in the GC log file.
```

Comment:
```suggestion
     * Return the estimated time of the first event in the GC log file.
```

Reply:
The time for first event is the time of the first event. The estimated time should maybe be renamed to startTime for the log